tests/DependencyInjection/ConfigurableRectorImportConfigCallsMergeTest.php

```diff
@@ -49,8 +49,8 @@ public function provideData(): Iterator
             __DIR__ . '/config/main_config_with_own_value.php', [
                 'old_2' => 'new_2',
                 'old_1' => 'new_1',
-                'old_4' => 'new_4',
                 'old_3' => 'new_3',
+                'old_4' => 'new_4',
```

Comment:
Moving structure means it got different result, which different with the original expected result on multiple call, /cc @ruudk as I can see you work on this before.

Reply:
The order should not matter, as configuration must be A → B unique items.
The important one here is that all values from all configs are included, not overriden just by the last ones.

Reply:
ok 👍